package.json

```diff
@@ -100,11 +100,12 @@
 		"@actions/github": "5.0.0",
 		"@apidevtools/json-schema-ref-parser": "11.6.4",
 		"@ariakit/test": "^0.4.2",
-		"@babel/core": "7.24.3",
-		"@babel/plugin-proposal-export-namespace-from": "7.18.9",
-		"@babel/plugin-syntax-jsx": "7.24.1",
-		"@babel/runtime-corejs3": "7.24.1",
-		"@babel/traverse": "7.24.1",
+		"@babel/core": "7.25.7",
+		"@babel/plugin-syntax-jsx": "7.25.7",
+		"@babel/plugin-transform-async-generator-functions": "7.25.7",
+		"@babel/plugin-transform-export-namespace-from": "7.25.7",
```

Comment:
The `@babel/plugin-proposal-export-namespace-from` and `@babel/plugin-proposal-async-generator-function` are deprecated, and their docs reference the new packages that replaces them:

- https://www.npmjs.com/package/@babel/plugin-proposal-export-namespace-from
- https://www.npmjs.com/package/@babel/plugin-proposal-async-generator-functions

> Both of these plugins (`async-generator-functions` and `export-namespace-from`) are part of the default preset (`@babel/preset-env`) we're using, so we should be able to remove them as explicit dependencies.

@tyxla just to make sure I understand, are you suggesting the removal of `@babel/plugin-transform-async-generator-functions` and `@babel/plugin-transform-export-namespace-from`?

Should we add an explicit dependency from `@babel/preset-env`, then ?

Reply:
I thought this might be related to deprecated warnings when running `npm ci` like this:

```
npm warn deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.
```

But I still see them on this branch. Not a blocker but it would be great to track those all down.

Reply:
Many babel plugins are referenced directly in the native test config:

https://github.com/WordPress/gutenberg/blob/6be7f8f9ecb8fdc0f1c69803507850a1781407bb/test/native/babel.config.js#L6-L16

Reply:
> @tyxla just to make sure I understand, are you suggesting the removal of @babel/plugin-transform-async-generator-functions and @babel/plugin-transform-export-namespace-from?

Ideally, yes, unless there's a good reason to keep them.

> Should we add an explicit dependency from @babel/preset-env, then ?

There should already be one in the `@wordpress/babel-preset-default` project. Worth verifying if all different type of tests use that preset though.

> Not a blocker but it would be great to track those all down.

I agree this isn't a blocker and can be done separately 👍 

Would be nice to check if we can ditch them for the react native tests and wherever else we're specifically referencing them.

Reply:
Yes, it's definitely a follow-up. I mostly wanted to raise it as I had some blurry memories that they standardized the naming to make it easier to distinguish that the proposal becomes the standard, so moving forward, it should be part of the `@babel/preset-env`. The Babel config for React Native might have some differences, so that needs to be double checked if it is still required there.

Reply:
I removed the `@babel/plugin-transform-async-generator-functions` and `@babel/plugin-transform-export-namespace-from dependencies` from `package.json`, the `package-lock.json` still includes them:

https://github.com/WordPress/gutenberg/pull/65949/commits/028275dfd23229f196aa369583d12e2243325adf

So, if all CI checks pass, I guess we're probably better off omitting them anyway, as they should be already included with the dependencies of `@wordpress/babel-preset-default`

If folks are ok with the current state of the PR, I'd like to stop here and get it merged to limit the scope.